packages/message-system/src/schema/config.schema.v1.json

```diff
@@ -282,14 +282,33 @@
                                 },
                                 "additionalProperties": false
                             },
+                            "feature": {
+                                "type": "array",
+                                "minItems": 1,
+                                "items": {
+                                    "title": "Feature",
+                                    "description": "Only used for 'feature' category. Feature flag can disable a feature for a specific version of a specific app.",
+                                    "type": "object",
+                                    "required": ["domain", "flag"],
+                                    "properties": {
+                                        "domain": {
```

Comment:
maybe it could be better named if we are not planning to implement some dynamic logic like super-context (`coins.*.receive`) but I don't want to use `key` (too generic?) and `domain` sounds good enough to me

Reply:
@tomasklim any proposals or `domain` sounds good enough to you too?